src/ncp/spinel.c

```diff
@@ -1271,6 +1271,10 @@ const char *spinel_prop_key_to_cstr(spinel_prop_key_t prop_key)
         ret = "PHY_CHAN_SUPPORTED";
         break;
 
+    case SPINEL_PROP_PHY_CHAN_PREFERRED:
```

Comment:
nit: We usually keep the order of `case` statements here match the order in `spinel.h` definition (make it easier to check if one is missing later). How about moving this after `SPINEL_PROP_PHY_PCAP_ENABLED`?